internal/pkg/template/templates/workloads/partials/cf/rollback-alarms.yml

```diff
@@ -0,0 +1,51 @@
+{{- if .DeploymentConfiguration.Rollback.CPUUtilization}}
+CPURollbackAlarm:
+  Metadata:
+    'aws:copilot:description': "A CloudWatch alarm associated with CPU utilization for deployment rollbacks"
+  Type: AWS::CloudWatch::Alarm
+  Properties:
+    AlarmDescription: "Roll back ECS service if CPU utilization is greater than or equal to {{.DeploymentConfiguration.Rollback.CPUUtilization}}% twice in 2 minutes."
+    AlarmName: {{.DeploymentConfiguration.Rollback.TruncateAlarmName .AppName .EnvName .WorkloadName "CopilotRollbackCPUAlarm"}}
+    Namespace: 'AWS/ECS'
+    Dimensions:
+      - Name: ClusterName
+        Value:
+          Fn::ImportValue:
+            !Sub '${AppName}-${EnvName}-ClusterId'
+      - Name: ServiceName
+        Value: !Select [2, !Split ["/", !Ref Service]]
+    MetricName: 'CPUUtilization'
+    ComparisonOperator: 'GreaterThanOrEqualToThreshold'
+    DatapointsToAlarm: 2
+    EvaluationPeriods: 2
+    Period: 60
```

Comment:
I'm slightly worried that 2 mins as a default setting isn't enough to judge if a deployment was successful. Would you mind reaching out to @nathanpeck to see what defaults he would like to see in a tool like Copilot and CDK extensions?

(don't block on this feedback) you can apply it retroactively in a separate PR.

My preference for a default would be:
```
    DatapointsToAlarm: 8
    EvaluationPeriods: 10
    Period: 60
```

Reply:
[pending]